src/pages/home/Home.page.tsx

```diff
@@ -49,6 +49,13 @@ const statData: StatData[] = [
     icon: HiOutlineMail,
     percentage: "30%",
   },
+  {
+    id: 3,
+    label: "Total de orçamentos",
+    score: 100,
```

Comment:
Tip - The label "Total de orçamentos" is in Portuguese. If the rest of your application's text is in a different language, you may want to ensure consistent language usage across the application. Otherwise, verify text consistency as per project requirements.